test/functional/page_objects/visualize_page.js

```diff
@@ -373,7 +373,7 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
     }
 
     async getGaugeValue() {
-      const elements = await find.allByCssSelector('visualize .chart svg');
+      const elements = await find.allByCssSelector('.visualize .chart svg');
```

Comment:
If you make the `data-test-subj` attribute change then this will benefit from being updated to `[data-test-subj="visualizationLoader"] .chart svg`